readme.md

```diff
@@ -145,6 +145,16 @@ You can pass an empty string to remove the suffix.
 
 For example, on macOS, the config file will be stored in the `~/Library/Preferences/foo-nodejs` directory, where `foo` is the `projectName`.
 
+#### watch
+
+type: `Boolean`<br>
+Default: `false`
+
+Sets a file watcher.
+
```

Comment:
The docs here needs to be improved. `Sets a file watcher.` is not a good or useful description. It also has no mention of triggering `onDidChange`.